pythonforandroid/bootstraps/sdl2/build/src/main/java/org/kivy/android/PythonUtil.java

```diff
@@ -45,6 +45,8 @@ protected static void addLibraryIfExists(ArrayList<String> libsList, String patt
         addLibraryIfExists(libsList, "crypto.*", libsDir);
         libsList.add("python2.7");
         libsList.add("python3.5m");
+        libsList.add("python3.6m");
+        libsList.add("python3.7m");
```

Comment:
Shouldn't theses changes added in all others bootstraps?

Reply:
The other bootstraps don't actually support python3 yet, I think it's best to leave that as a separate task rather than delay merging.